### DISCUSSION TITLE
SystemError: initialization of _internal failed without raising an exception

### DISCUSSION THREAD

**Discussion by gaggiag** [openai/whisper#1103] [Q&A] [answered]
I am trying to install whisper, and everything seemed to go very smoothly without any errors during installation.
However, I get this error message when I try to import whisper into python:

`SystemError: initialization of _internal failed without raising an exception`

Searching the web, it seems like a problem usually arises with numpy 1.24, but the numpy that was installed was 1.23.5. So it couldn't be it.
Any ideas how to solve this?

I am using MacOS Monterey, my python version is 3.10.4.

**Comment by brianjorden** [comment, marked as answer]
Had the same issue, but seem to have gotten past it.

```
pip uninstall numpy
pip uninstall numba
python3 -m pip install numba
```

Seems to be a dependency version issue. When you reinstall the `numba` package, it pulls in the right version of `numpy`.

WARNING: I haven't done much with python, so I'm not concerned about screwing up my environment. This worked for me, but may break something else for others.

**Reply by gaggiag** [reply]
Thanks! Unfortunately it didn't work for me. I still get the same error.

**Reply by gaggiag** [reply]
Oh, never mind - I had to restart the kernel and then it worked. Thanks!

**Reply by emiedeltares** [reply]
Worked for me too.  Thanks :)

**Reply by ArjunS07** [reply]
Fantastic, it works, thank you!

**Reply by paperadio** [reply]
It really works, Thank you!

**Reply by pwernette** [reply]
Worked great for me, only I had to also run `python -m pip install llvnlite --ignore-installed` before `python -m pip install numba`.

**Reply by GoingBin** [reply]
you are ringht!

**Comment by serg06** [comment]
Identical but simpler solution:

```bash
pip install -U numba
```

**Reply by IsabeleVitorio** [reply]
Thanks!

**Comment by eromoe** [comment]
[brianjorden](https://github.com/brianjorden) is right, and be careful if you are using conda env. There may be two numpy installed ... I unstalll twice remove two different version numpy . Then new installation of numba work without error.
